### ISSUE TITLE
Galaxy pending

### ISSUE THREAD

**Issue by mcapuccini** [phnmnl/cloud-deploy-kubenow#35]
Sometimes Galaxy remains in `Pending` phase. Probably due to lack of resources, we may want to decrease Jupyter resources, so it will always find space.

```
NAME                       READY     STATUS    RESTARTS   AGE
galaxy-k8s-3knpq           0/1       Pending   0          19m
gluster-server-8662c       1/1       Running   0          20m
gluster-server-d9r64       1/1       Running   0          20m
gluster-server-p5lsv       1/1       Running   0          20m
jupyter-1684481266-ff7tp   1/1       Running   0          20m
luigi-3395107068-kmft4     1/1       Running   0          19m
postgresql-server-jdckp    0/1       Pending   0          19m
```

**Comment by andersla** [comment]
jupyter was only 200m, Luigi was 1cpu - Should I change luigi to 200m ?

**Comment by mcapuccini** [comment]
Yep, Luigi should be ok with 200m

**Comment by andersla** [comment]
Fixed by #40
